src/registrar/models/domain_request.py

```diff
@@ -12,6 +12,7 @@
 from registrar.models.federal_agency import FederalAgency
 from registrar.models.utility.generic_helper import CreateOrUpdateOrganizationTypeHelper
 from registrar.utility.errors import FSMDomainRequestError, FSMErrorCodes
+from registrar.utility.constants import BranchChoices
```

Comment:
Good call on updating the rest of our models that reference branch choices with the new constant ref!